source/server/http/admin.cc

```diff
@@ -1074,13 +1074,30 @@ std::string AdminImpl::runtimeAsJson(
   return strbuf.GetString();
 }
 
+bool AdminImpl::isFormUrlEncoded(const Http::HeaderEntry* content_type) const {
+  if (content_type == nullptr) {
+    return false;
+  }
+
+  return content_type->value().getStringView() == "application/x-www-form-urlencoded";
```

Comment:
add "application/x-www-form-urlencoded" to well_known_names.cc like other http header values.

Reply:
you meant here https://github.com/envoyproxy/envoy/blob/master/source/common/http/headers.h#L133 ?

Reply:
yes that's what I meant :)

Reply:
Updated.